api/app/settings/common.py

```diff
@@ -173,21 +173,39 @@
         ),
     }
     REPLICA_DATABASE_URLS_DELIMITER = env("REPLICA_DATABASE_URLS_DELIMITER", ",")
-    REPLICA_DATABASE_URLS = env.list(
-        "REPLICA_DATABASE_URLS", default=[], delimiter=REPLICA_DATABASE_URLS_DELIMITER
+    # Allow specifying replica database URLs as a JSON array to avoid delimiter conflicts
+    REPLICA_DATABASE_URLS_JSON = json.loads(
+        env.str("REPLICA_DATABASE_URLS_JSON", "null")
     )
+
+    if REPLICA_DATABASE_URLS_JSON:
```

Comment:
nit: `REPLICA_DATABASE_URLS_JSON` is not JSON but python here.. Maybe we can change it something like:

```
REPLICA_DATABASE_URLS_JSON = env.str("REPLICA_DATABASE_URLS_JSON")
if REPLICA_DATABASE_URLS_JSON:
  REPLICA_DATABASE_URLS= json.loads(REPLICA_DATABASE_URLS)

```